src/Twilio.Api.Silverlight/Sms.Async.cs

```diff
@@ -27,7 +27,7 @@ public virtual void GetSmsMessage(string smsMessageSid, Action<SMSMessage> callb
 		/// </summary>
 		/// <param name="callback">Method to call upon successful completion</param>
         [Obsolete("Use ListMessages instead.")]

```

Comment:
System.Obsolete directly references the System namespace while Obsolete implies that file is using the System namespace. The are referring to the same Obsolete but System.Obsolete doesn't import System globally